### PR TITLE
Add compatibility with ElasticSearch 2.x

### PR DESCRIPTION
**Pull request by kane-c**
ES 2.x does not have a `status` in the this API call any more. In that scenario, just fall back to the HTTP status code.

### CODE REVIEW

File: forklift/services/elasticsearch.py [hunk 1]
```diff
@@ -104,7 +104,11 @@ def check_available(self):
             url = replace_part(url, path='')
             es_response = urllib.request.urlopen(url.geturl())
             es_status = json.loads(es_response.read().decode())
-            if es_status['status'] != 200:
+            try:
```
Comment (jscn) [inline forklift/services/elasticsearch.py]: I feel like this is starting to get a little long, did you think about splitting the status check out into a new method?

What would you thins about inlining getting the status, too: `status = es_status.get('status', es_response.status)` ?

**Comment by jscn** [comment]
Looks like there are some problems with the tests :(

**Comment by kane-c** [comment]
Yeah, PyLint 1.5 is out now which includes import order checking. Currently fixing things up...

File: forklift/services/elasticsearch.py (hunk 1 above)
Comment (kane-c) [inline forklift/services/elasticsearch.py]: Yeah, I'll inline the status. Much cleaner.

**Comment by jscn** [comment]
:+1:

**Pull request merged**